core/templates/pages/exploration-editor-page/changes-in-human-readable-form/changes-in-human-readable-form.component.ts

```diff
@@ -18,17 +18,10 @@
 
```

Comment:
Should we replace `Directive` with `Component` in the fileoverview? (Not sure whether we fo this!)

Reply:
I think it's ok, done!